src/DaPigGuy/PiggyCustomEnchants/enchants/armor/AttackerDeterrentEnchant.php

```diff
@@ -50,14 +50,6 @@ public function __construct(PiggyCustomEnchants $plugin, int $id, string $name,
         parent::__construct($plugin, $id, $rarity);
     }
 
-    /**
-     * @return array
-     */
-    public function getReagent(): array
-    {
-        return [EntityDamageByEntityEvent::class];
-    }
-
```

Comment:
This class should ``use ReactiveTrait`` instead of extending ReactiveEnchantment
> Couldn't comment line 24, so I commented this line